CMakeLists.txt

```diff
@@ -277,6 +277,10 @@ endif()
 if(GINKGO_BUILD_HWLOC)
     find_package(HWLOC 2.1) # No need for QUIET as we ship FindHWLOC
 endif()
+if(GINKGO_BUILD_PAPI_SDE)
+    # No need for QUIET as we ship FindPAPI
+    find_package(PAPI OPTIONAL_COMPONENTS sde)
+endif()
```

Comment:
Ah, I understand the problem now. Similarly to hwloc, my understanding is `GINKGO_BUILD_PAPI_SDE` is only for the user to tell us whether he wants to build Ginkgo with PAPI SDE support or not. Then if it is set, we first try to find a suitable version on the system, otherwise, we build it ourselves. Maybe the option naming could change if it's not clear enough, though that would imply reviewing the hwloc option as well.

Reply:
Ok, now I understand that part. Perhaps `GINKGO_WITH_PAPI_SDE` would be better?